version.php

```diff
@@ -29,8 +29,8 @@
     die('Direct access to this script is forbidden.'); // It must be included from a Moodle page.
 }
 
-$plugin->version   = 2020112000;      // The current plugin version (Date: YYYYMMDDXX).
-$plugin->release   = 2020112000;      // Same as version
+$plugin->version   = 2020113000;      // The current plugin version (Date: YYYYMMDDXX).
```

Comment:
so 2020112000 semantically means 2020 / 11 /20 + 00 where 20th nov is the fork date. So the for the older versions we only want to bump the sub-version, ie:

2020112000 -> 2020112001